ios/FluentUI/People Picker/AvatarView.swift

```diff
@@ -493,8 +524,42 @@ open class AvatarView: UIView {
         }
     }
 
+    private func updateImageViewWithPersonIcon(isSignedOut: Bool = false) {
```

Comment:
Can we add some documentation to these functions? I'm not quite sure what to expect behavior-wise, and how the input impacts what actually happens.